gpsnavi: Update permissions

Add the new display and audio permissions required with the change to
running as non-root.

Upstream-Status: Inappropriate [no upstream]

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/agl/config.xml b/agl/config.xml
index 9d4c0ca..44de94a 100755
--- a/agl/config.xml
+++ b/agl/config.xml
@@ -8,6 +8,8 @@
   <feature name="urn:AGL:widget:required-permission">
     <param name="urn:AGL:permission::public:no-htdocs" value="required" />
     <param name="http://tizen.org/privilege/internal/dbus" value="required" />
+    <param name="urn:AGL:permission::public:display" value="required" />
+    <param name="urn:AGL:permission::public:audio" value="required" />
   </feature>
   <feature name="urn:AGL:widget:required-api">
     <param name="homescreen" value="ws" />
